src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -62,6 +62,7 @@
 							:show-action-button="true"
 							:show-tags="enableLoanTags"
 							:in-grid="true"
+							:category-page-name="loanChannelName"
```

Comment:
This needs to be passed in `HelpmeChooseWrapper` as well, otherwise you get a tag that matches the categories, like in Agriculture